cmd/create.go

```diff
@@ -410,8 +410,8 @@ func createHost(provider, name, region, zone, projectID, userData, inletsPort st
 	} else if provider == "civo" {
 		return &provision.BasicHost{
 			Name:       name,
-			OS:         "811a8dfb-8202-49ad-b1ef-1e6320b20497",
-			Plan:       "g2.small",
+			OS:         "12745392-15c7-4140-925d-441fe7ae57fd",
+			Plan:       "g3.small",
```

Comment:
Did they delete the cheaper g2 small instances? What's the cost of the g3?

Reply:
![image](https://user-images.githubusercontent.com/38325136/124720863-dfa81880-df08-11eb-89e0-eba85704cd11.png)
They dont offer the generation 2 sizes anymore...

![image](https://user-images.githubusercontent.com/38325136/124721047-0e25f380-df09-11eb-8ae0-46f0b6d7904d.png)
i was not sure if g3.xsmall is okay from the spec...

Reply:
@alexellis what was the spec of the g2.small? 1CPU/1GB or 2GB?

Reply:
From memory it was a 5 USD variant, so let's use the Extra Small variant please 👍

Reply:
Okay i found a old list. So i Think g3.small is fine...
![image](https://user-images.githubusercontent.com/38325136/124724050-ed12d200-df0b-11eb-9247-4fe79c481439.png)

Reply:
ok @alexellis, i change it to g3.xsmall